integration/src/main/java/scripts/testscripts/ListProviders.java

```diff
@@ -0,0 +1,34 @@
+package scripts.testscripts;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.externalcreds.api.OidcApi;
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import com.google.api.client.http.HttpStatusCodes;
+import lombok.extern.slf4j.Slf4j;
+import scripts.utils.ClientTestUtils;
+
+@Slf4j
+public class ListProviders extends TestScript {
+
+  @Override
+  public void userJourney(TestUserSpecification testUser) throws Exception {
+    log.info("Checking the version endpoint.");
+    var apiClient = ClientTestUtils.getClientWithoutAccessToken(server);
+    var oidcApi = new OidcApi(apiClient);
```

Comment:
The `OidcApi` usually requires authentication through the proxy but it seems the proxy isn't used when running integration or perf tests - so we get by here without authenticating. Just want to confirm - does that sound right to others?

Reply:
After sleeping on this, I think it makes sense that there's no proxy to require auth locally. The next step will be to run these tests against perf - and I'm pretty sure that the proxy _will_ require authentication in that case, so I'll probably need to add an access token once we set that up (in my next PR if that's okay with reviewers 😄).